weaviate/exceptions.py

```diff
@@ -29,6 +28,10 @@ def __init__(self, message: str = ""):
         super().__init__(message)
 
 
+class RequestsConnectionError(ConnectionError):
```

Comment:
De we need this one?

Reply:
Also note that the `ConnectionError` is the build-in one not the requests one.

Reply:
No, leftover from rebase